Scheduler/feature_scheduler/maintel/fbs_config_fieldsurvey.py

```diff
@@ -0,0 +1,120 @@
+# This file is part of ts_config_ocs.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from lsst.ts.fbs.utils.data.field_survey_centers import get_sv_fields
+from lsst.ts.fbs.utils.maintel.basis_functions import get_basis_functions_field_survey
+from lsst.ts.fbs.utils.maintel.detailers import get_detailers_field_survey
+from lsst.ts.fbs.utils.maintel.make_fieldsurvey_scheduler import (
+    MakeFieldSurveyScheduler,
+)
+
+
+def get_scheduler():
+    """Construct feature based scheduler.
+
+    Returns
+    -------
+    nside : int
+        Healpix map resolution.
+    scheduler : Core_scheduler
+        Feature based scheduler.
+    """
+
+    nside = 32
+
+    make_scheduler = MakeFieldSurveyScheduler(nside=nside, ntiers=1)
+
+    nvisits = {"u": 20, "g": 20, "r": 20, "i": 20, "z": 20, "y": 20}
+    sequence = "ugrizy"
+    # exposure time in seconds
+    exptimes = {"u": 38, "g": 30, "r": 30, "i": 30, "z": 30, "y": 30}
+    # 1 --> single 30 second exposure
+    nexps = {"u": 1, "g": 1, "r": 1, "i": 1, "z": 1, "y": 1}
+
+    wind_speed_maximum = 13.0  # maximum direct wind in m/s
+
+    field_survey_kwargs = {
+        "nvisits": nvisits,
+        "sequence": sequence,
+        "exptimes": exptimes,
+        "nexps": nexps,
+    }
+
+    # This might move to be part of a separate function
+    basis_functions = get_basis_functions_field_survey(
+        nside=nside,
+        wind_speed_maximum=wind_speed_maximum,
+    )
```

Comment:
I set the default to 40 deg in the default set of basis functions.

Reply:
Wait - why do you need two AltAzShadowMaskBasisFunctions? 
The new version of this picks up `tel_alt_limit` from the Conditions and uses that as a mask. So if Tiago sets this in the Conditions, we're good to go with only one AltAzShadowMaskBasisFunction.
(it could also be set via `sky_alt_limit` if `tel_alt_limit` is hard-coded to hard-constraints on the telescope pointing).

Reply:
My idea of having two is that one stays in the base configuration for the field surveys (ts_fbs_utils) to provide the base set of nominal constraints that we expect when the system is fully functional and another is here at the ts_config_ocs level to provide flexible constraints when needed. Right now, the AltAz constraints are changing pretty rapidly (day to day) so I didn't want to bury that down in ts_fbs_utils. 

I didn't know about the new feature to pick the tel_alt_limit up from the Conditions, so I think that alone would work if we are always updating that, but I don't think we are since the system constraints are changing so quickly. Sometimes we don't get the report until the afternoon so having an option in ts_config_ocs to mask out different regions of the sky on the fly will be very useful for the next few weeks.

Reply:
Also, I just saw Tiago propose an alternative on the PR in ts_fbs_util, which is just to expose the parameters for the AltAzShadowMaskBasisFunction to the user...I think that's cleaner than having two basis functions, and we could even leave it with a default set for our "nominal" constraints.

Reply:
Yes if it’s hard to update the tel-alt-limit in the conditions, it should definitely just be set by the user in ts-FBs-utils.

Reply:
Now passing in min_alt and max_alt from ts_config_ocs level.